src/CommonITILSatisfaction.php

```diff
@@ -155,11 +155,21 @@ public function showSatisactionForm($item)
                 $this->fields["satisfaction"] = $default_rate;
             }
             $max_rate = Entity::getUsedConfig('inquest_config' . $config_suffix, $item->fields['entities_id'], 'inquest_max_rate' . $config_suffix);
+            $duration = (int) Entity::getUsedConfig('inquest_duration', $item->fields['entities_id']);
+            $date2 = strtotime($this->fields['date_begin']);
+            $expired = true;
+            if (
+                ($duration === 0)
+                || (time() - $date2) <= $duration * DAY_TIMESTAMP
+            ) {
+                $expired = false;
+            }
```

Comment:
Could be simplified.
```suggestion
            $expired = $duration !== 0 && (time() - strtotime($this->fields['date_begin'])) > $duration * DAY_TIMESTAMP;
```